iphone/Classes/TiMediaAudioPlayerProxy.m

```diff
@@ -1,14 +1,16 @@
 /**
  * Appcelerator Titanium Mobile
- * Copyright (c) 2009-2015 by Appcelerator, Inc. All Rights Reserved.
+ * Copyright (c) 2009-2016 by Appcelerator, Inc. All Rights Reserved.
  * Licensed under the terms of the Apache Public License
  * Please see the LICENSE included with this distribution for details.
  */
-#ifdef USE_TI_MEDIA
+#ifdef USE_TI_MEDIAAUDIOPLAYER
 
 #import "TiMediaAudioPlayerProxy.h"
 #import "TiUtils.h"
+#ifdef USE_TI_MEDIAAUDIOSESSION
```

Comment:
Using KitchenSink, the build fails because of this. Apparently, `USE_TI_MEDIAAUDIOSESSION` will not be generated in defines.h when building to device. commenting this out makes it build normally.